packages/graph/hash_graph/.config/config.toml

```diff
@@ -0,0 +1,102 @@
+[target.'cfg(all())']
+rustflags = [
+    "-Wfuture_incompatible",
+    "-Wnonstandard_style",
+    # "-Wrust_2018_idioms",
```

Comment:
Why is this commented out vs. not just removed? Is there a 2021 equivalent we'd rather use that isn't out yet?

Reply:
`rust_2018_idioms` are idiomatic practices, which were introduced with the 2018 edition, those are not outdated. There are a few lints I like to disable, but I didn't want to address those in this PR. The 2021 edition AFAIK does not have such a lint.